kubernetes/fineractmysql-deployment.yaml

```diff
@@ -0,0 +1,101 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements. See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership. The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+apiVersion: v1
+kind: Service
+metadata:
+  creationTimestamp: null
+  labels:
+    app: fineractmysql
+  name: fineractmysql
+spec:
+  ports:
+  - name: "3306"
+    port: 3306
+    targetPort: 3306
+  selector:
+    app: fineractmysql
+status:
+  loadBalancer: {}
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
```

Comment:
Is this complete?
Correct me if I am wrong, I don't see values such as `storageClassName`, `PersistentVolume` etc.

Reply:
> Correct me if I am wrong, I don't see values such as `storageClassName`, `PersistentVolume` etc.

Yes, from what I saw declaring and deploying a PVC automatically creates a persistent storage

Reply:
Sorry, if this question seems redundant. 
I just want to be sure. 
Is dynamic provisioning enabled by default in minikube.
https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/ 
I used the same yaml as yours and tried volume mounting a pod. 
It won't work unless this is enabled and that too with `storageClassName`.

If you are sure about this thing not causing any issues, then that's a go-ahead from me.
```
---
apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  creationTimestamp: null
  labels:
    app: fineractmysql-claim0
  name: fineractmysql-claim0
spec:
  accessModes:
  - ReadOnlyMany
  resources:
    requests:
      storage: 100Mi
status: {}